src/tests/libclient/timing/onofftiming/tst_onofftiming.cpp

```diff
@@ -0,0 +1,25 @@
+#include <QtTest>
+
+#include <timing/onofftiming.h>_>
+
+class TestOnOffTiming : public QObject
+{
+    Q_OBJECT
+
+private slots:
+    void nextRuns(){
```

Comment:
Even though this is @monstermunchkin's area, please move the opening bracket to the next line.